src/lib/core/media-trigger/media-trigger.ts

```diff
@@ -151,7 +149,7 @@ export class MediaTrigger {
   private forceRegistryMatches(queries: string[], matches: boolean) {
     const registry = new Map<string, MediaQueryList>();
     queries.forEach(query => {
-      registry.set(query, {matches: matches} as MediaQueryList);
+      registry.set(query, {matches: matches, removeListener: (_) => {}} as MediaQueryList);
```

Comment:
This shouldn't be necessary anymore, given that removal functions are only registered when creating a real media query list.